ansible/ext-inventory.yml

```diff
@@ -0,0 +1,65 @@
+# This playbook updates data at ooni-sysadmin.git/ext/
+---
+- hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - name: get git-root
+      command: git rev-parse --show-toplevel
+      register: extroot
+      changed_when: false
+
+    - set_fact:
+        extroot: '{{ extroot.stdout }}/ext'
+        socks5: '{{ 32767 | random(1024) }}'
+        jumphost: '{{ namecheap_jumphosts | random }}'
```

Comment:
I would prefer if there was one host specifically configured to be a jumphost, perhaps we should add such a role to the playbook.